bootstrap/kubeadm/api/v1alpha3/suite_test.go

```diff
@@ -56,7 +56,10 @@ func TestAPIs(t *testing.T) {
 var _ = BeforeSuite(func(done Done) {
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join("..", "..", "..", "..", "config", "crd", "bases")},
+		CRDDirectoryPaths: []string{
+			filepath.Join("..", "..", "..", "..", "config", "crd", "bases"),
+			filepath.Join("..", "..", "..", "..", "config", "bootstrap", "crd", "bases"),
```

Comment:
Nit, likely should add the `controlplane` provider here as well, but that can also be tackled when it's needed.

Reply:
fixed this and the other one in amended commit.